libs/wingc/src/type_check.rs

```diff
@@ -2544,8 +2544,8 @@ impl<'a> TypeChecker<'a> {
 		let initialized_fields = visit_init.fields;
 
 		for field in fields.iter() {
-			// inflight or static fields cannot be initialized in the initializer
-			if field.phase == Phase::Inflight || field.is_static {
+			// static fields cannot be initialized in the initializer
+			if field.is_static {
```

Comment:
Undo

Reply:
I understand now, sorry, I don't notice the `inflight init` constructor 

I will make some changes to capture the specific phase initialization

Reply:
@Chriscbr Question: inflight var fields needs to be initialized ?

Reply:
@Chriscbr 

Do the `inflight var` fields need to be initialized?

Reply:
Yes 👍